### ISSUE TITLE
Using `pyam.plotting.PYAM_COLORS` effectively

### ISSUE THREAD

**Issue by danielhuppmann** [IAMconsortium/pyam#568]
I'm trying to use the `PYAM_COLORS` feature for preparing plots of AR6, and I'm wondering how to efficiently use the pre-defined IPCC-colors.

As a first step, I added this snippet at the start of a notebook, and then make sure that the scenarios use exact names like `AR6-SSP1-1.9`.

```python
rc = pyam.run_control()

for key, value in pyam.plotting.PYAM_COLORS.items():
    rc.update(dict(color=dict(scenario={key: value})))

df.plot(color="scenario")
```

Is that the intended usage of this feature? Would it make sense to add a utility function like `use_pyam_colors()` to load the pre-defined colors into the RunControl?

**Comment by danielhuppmann** [comment]
@gidden, any thoughts?

**Comment by gidden** [comment]
I have used it as follows in my own wg1 plotting repo

1. define the color mapping you want in a [yaml file](https://github.com/gidden/ar6-wg1-ch6-emissions/blob/master/plotting.yaml)

- this can use the PYAM_COLORS directly

2. [Read in](https://github.com/gidden/ar6-wg1-ch6-emissions/blob/master/plots.py#L13) that YAML file at the start of the plotting code/routine

I think the corner case that you're hitting is that the scenarios themselves have the same name as their 'color identifier'. 

When I first wrote this up, I wanted to be as specific as possible, i.e. "this is the color of SSP1 RCP2.6 used in AR6" - hence the color identifier is `AR6-SSP1-2.6`.

I think it would be better to have the scenario name be `SSP1-2.6` (as I do in the YAML file), and then update the run control for it to be colored using the color identifier `AR6-SSP1-2.6`.

If we were to do this 'automagically', it would be akin to naming a scenario "blue" and then expecting it to be blue in any subsequent plot. I'm not sure that's what we want. But happy to continue the conversation here.
